src/interfaces/Quiz.d.ts

```diff
@@ -0,0 +1,13 @@
+import { PostAPIBase } from './PostAPI';
+
+export interface QuizContent {
+  question: string;
+  des: string;
+  tag: string[];
+  difficulty: number;
+  importance: number;
+  answerType: 't/f' | 'numberType' | 'stringType';
```

Comment:
answerType에 문자타입이 아니라, 정답유형의 의미론적으로 담을 수 있게 표현하는게 더 좋을 것 같아요. 나중에 코드형식, 긴문장(에세이) 형식등 추가 고려할 수 있으니!
```suggestion
  answerType: 'trueOrFalse' | 'multipleChoice | 'shortAnswer';
```

Reply:
와 너무 좋은 변수명이네요.....!!